keeper/src/main/scala/zio/keeper/transport/package.scala

```diff
@@ -1,17 +1,28 @@
 package zio.keeper
 
+import zio.keeper.transport.Channel._
 import zio.nio.core.SocketAddress
 import zio.{ Has, ZIO, ZManaged }
-import zio.keeper.transport.Channel._
 
 package object transport {
 
   type Transport = Has[Transport.Service]
 
-  def bind(
+  def bind[R](
```

Comment:
I think `R <: Transport` tends to infer better

Reply:
done